<?xml version="1.0"?>
<launch>
  <arg name="veh" default="wamv"/>

  <!-- Joystick Driver -->
  <node ns="$(arg veh)" pkg="joy" type="joy_node" name="joy">
    <param name="dev" value="/dev/input/js0"/>
  </node>


  <!-- Convert Twist messages to Drive messages -->
  <node ns="$(arg veh)" pkg="vrx_gazebo" type="joystick2thrust.py" name="joystick2thrust" output="screen">
    <remap from="left_cmd" to="/$(arg veh)/thrusters/left_thrust_cmd"/>
    <remap from="right_cmd" to="/$(arg veh)/thrusters/right_thrust_cmd"/>
    <remap from="left_lateral_cmd" to="/$(arg veh)/thrusters/left_lateral_thrust_cmd"/>
    <remap from="right_lateral_cmd" to="/$(arg veh)/thrusters/right_lateral_thrust_cmd"/>
  </node>
</launch>